x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.tsx

```diff
@@ -13,10 +13,12 @@ import {
   Query,
   TimefilterContract,
   TimeRange,
+  IndexPattern,
 } from 'src/plugins/data/public';
 import { ExecutionContextSearch } from 'src/plugins/expressions';
 
 import { Subscription } from 'rxjs';
+import { Ast } from '@kbn/interpreter/target/common';
```

Comment:
`target` should not be imported:

```suggestion
import { Ast } from '@kbn/interpreter/common';
```

Reply:
Fixed all of the instances of importing from target in the next version.